pyproject.toml

```diff
@@ -23,7 +23,7 @@ pydantic-core = "^2.18.0"
 fastapi = { version = "^0.109.2", optional = true }
 redis = { version = "^5.0.1", optional = true }
 diskcache = { version = "^5.6.3", optional = true }
-pandas = { version = "^2.2.0", optional = true }
+pandas = { version = ">=0.24.2,<2.2.0", optional = true }
```

Comment:
Had to mess with versions for now to get `ibm-watsonx-ai` to work - will try to resolve better later